lib/rspec/rails/adapters.rb

```diff
@@ -15,11 +15,31 @@ module Rails
       # loaded.
       Assertions = Minitest::Assertions
     elsif RUBY_VERSION >= '2.2.0'
-      # Minitest has been removed from ruby core. However, we must be on an
-      # old Rails version and must load the Minitest 4.x gem
-      gem 'minitest' if defined?(Kernel.gem)
-      require 'minitest/unit'
-      Assertions = MiniTest::Assertions
+      # Minitest / TestUnit has been removed from ruby core. However, we are
+      # on an old Rails version and must load the appropriate gem
+      if ::Rails::VERSION::STRING >= '4.0.0'
+        # ActiveSupport 4.0.x has the minitest '~> 4.2' gem as a dependency
+        gem 'minitest' if defined?(Kernel.gem)
+        require 'minitest/unit'
+        Assertions = MiniTest::Assertions
+      elsif ::Rails::VERSION::STRING >= '3.2.21'
```

Comment:
3-part versions compare differently than their string form does, particular when you have parts with multiple digits.  I think it's OK here because the 2-digit part is the last part...but thought I'd bring it up as something to be aware of and careful of.